### PR TITLE
exam

### PR DESCRIPTION
**Pull request by Dyd-sull**

### CODE REVIEW

File: src/views/IssueItem.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <h3 class="display-2 mt-3 mb-1">
+          {{ issue.title }} <span class="text-grey">#{{ issue.number }}</span>
+        </h3>
+
+        <h4>Opened by {{ issue.user.login }}  · <span class="text-grey">{{ issue.comments }} Comments</span></h4>
+
+        <br/>
+        <v-card>
+          <v-card-title primary-title>
+              <div v-html="issue.body" />
```
Comment (rapi) [inline src/views/IssueItem.vue:15]: v-html is not secure

File: src/views/IssueItem.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+  },
+  props: {
+    id: {
+      type: [Number, String],
+      required: true
+    }
+  },
+  created () {
+    axios.get('/issues/' + this.id)
```
Comment (rapi) [inline src/views/IssueItem.vue:40]: put in in differnet api function

File: src/views/IssuesList.vue [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import axios from '@/plugins/axios'
+export default {
+  name: 'IssuesList',
+  data () {
+    return {
+      issues: []
+    }
+  },
+  created () {
+    axios.get('/issues?per_page=10')
```
Comment (rapi) [inline src/views/IssuesList.vue:38]: dont use string like the parameter, use object

File: src/components/layouts/CommonLayout.vue [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'CommonLayout'
+}
+</script>
+
+<style scoped>
```
Comment (rapi) [inline src/components/layouts/CommonLayout.vue:13]: Dont use empty tags

**Review by rapi — COMMENTED** [review]
Eslint problem
Api funciton in different folder
dont use v-html

File: .eslintrc.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true
+  },
+  'extends': [
+    'plugin:vue/essential',
```
Comment (gregurco) [inline .eslintrc.js:7]: better to use `recommended` set

File: src/router/router.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  mode: 'history',
+  base: process.env.BASE_URL,
+  routes: [
+    {
+      path: '',
+      component: CommonLayout,
+      children: [
+        {
+          path: '',
+          name: 'issueslist',
```
Comment (gregurco) [inline src/router/router.js:19]: `'issues-list'`

**Pull request merged**